### DISCUSSION TITLE
Help with on_member_update()

### DISCUSSION THREAD

**Discussion by Fizzxx** [Pycord-Development/pycord#461] [Q&A] [answered]
I recently switched over from using the latest discord.py , and on_member_update does not trigger for changes in members' status or activity (such as streaming). It does, however, trigger for other events, such as adding/removing roles and changes in status and activity for the bot itself. I made sure I have intents set up properly, and even went back to using discord.py and it works using that.

Does anyone have any ideas what might be going on?
Here's the simplest bit of code that models my issue:

```

import discord
from discord.ext import commands

token = "token"
intents = discord.Intents.all()
the_bot = commands.Bot(command_prefix="/", intents=intents)


class MemberUpdateCog(commands.Cog):
    def __init__(self, bot):
        self.bot = bot

    @commands.Cog.listener()
    async def on_ready(self):  # <-- this works fine, and only included to demonstrate that the cog works fine
        print('[BOT READY]') 

    @commands.Cog.listener()
    async def on_member_update(self, before, after):  # <-- this does not work for status/activity changes, but does for roles and other changes
        print('[member updated]')  


def setup(bot):
    bot.add_cog(MemberUpdateCog(bot))


setup(the_bot)
the_bot.run(token)
```

**Comment by Quintenvw** [comment]
I have noticed this issue too.

**Comment by middledot** [comment, marked as answer]
Presence updates now go through on_presence_update

**Reply by Fizzxx** [reply]
Beautiful. Thank you so much!
